app/metrics.yaml

```diff
@@ -540,6 +540,29 @@ events:
     metadata:
       tags:
         - Sharing
+  share_to_app:
+    type: event
+    description: |
+      A user selected an app to share to from the share sheet
+    extra_keys:
+      app_package:
+        description: |
+          Package identifier for app or action the user selected
+          (ex. 'org.mozilla.focus' or 'com.android.bluetooth')
```

Comment:
I'm curious to hear from a data reviewer if reporting the selected app is kosher.

Reply:
Good catch.  I will confirm.

Reply:
I will have to escalate this.  Since we're collecting the package ID of third party apps, this is considered sensitive.